src/edu/stanford/nlp/sempre/Derivation.java

```diff
@@ -62,7 +81,7 @@
 
   // Information for scoring
   private final FeatureVector localFeatureVector;  // Features
-  double score = Double.NaN;  // Weighted combination of features
+  public double score = Double.NaN;  // Weighted combination of features
```

Comment:
I don't see where you need to write to this.  Use `getScore` if you just need to read from it.